assets/javascripts/discourse/helpers/tonable-emoji-title.js

```diff
@@ -0,0 +1,9 @@
+import { registerUnbound } from "discourse-common/lib/helpers";
+
+registerUnbound("tonable-emoji-title", function (emoji, scale) {
```

Comment:
Minor nit but maybe this should be called `diversityScale` or something since we are passing `chatEmojiReactionStore.diversity` as the arg

Reply:
Yep agreed, quickly thought of this and should have done it :+1: